lib/components/BuilderUI.jsx

```diff
@@ -235,6 +240,41 @@ const getPrizePoolDetails = (params, signer, chainId) => {
   }
 }
 
+/**
+ * Call proper builder fn based on selected prize pool type
+ * @param {*} prizePoolType
+ * @param {*} builderContract
+ * @param {*} prizePoolConfig
+ * @param {*} multipleRandomWinnersConfig
+ */
+const createPools = async (
+  prizePoolType,
+  builderContract,
+  prizePoolConfig,
+  multipleRandomWinnersConfig
+) => {
+  const gasLimit = 3000000
+
+  switch (prizePoolType) {
+    case PRIZE_POOL_TYPE.compound: {
+      return await builderContract.createCompoundMultipleWinners(
+        prizePoolConfig,
+        multipleRandomWinnersConfig,
+        TICKET_DECIMALS,
+        { gasLimit }
```

Comment:
We should test if this one still needs to be a 3000000 gas limit (it's a lot!)

Reply:
I just got about 1200000 on Rinkeby (1.2 million). Yet to try it on mainnet but yeah, I think we can bring this down as well.